### DISCUSSION TITLE
Suggestion: add a player preset for JRiver Media Center

### DISCUSSION THREAD

**Discussion by DrewNeilson** [streamlink/streamlink-twitch-gui#905] [General]
I use JRiver Media Center, and in Streamlink Twitch GUI's settings --> player, I've selected:
- For preset, I've selected "no preset"
- Video player: C:\Program Files\J River\Media Center 29\MC29.exe
--> Unfortunately, JRiver Media Center's .exe is "MC" plus the version number, and so whenever a new version of Media Center is released, I have to change this, for example from MC28.exe to MC29.exe. Not a big problem, but perhaps if Streamlink Twitch GUI had a profile for JRiver Media Center, I wouldn't have to specify Media Center's executable's name and location.
- Parameters: none
- In addition, if Streamlink Twitch GUI had a profile for JRiver Media Center, maybe it'd have options for Media Center like what it has for other players.
- I don't know exactly how many people use JRiver Media Center to watch Twitch streams, but many home theater enthusiasts use it to watch movies, TV shows, and other content, and/or to listen to music, and I figure that some of them also use it to watch Twitch. If Streamlink Twitch GUI had a built-in profile for JRiver Media Center, it'd help them.

I'm not yet familiar with how to create this profile myself, but if that is possible, I will do so, and I will contribute it here if that is possible.

**Comment by bastimeyer** [comment]
I'm not going to create player profiles for proprietary software which requires having a paid license.

Profiles are also merely a collection of known executable names, fallback paths (in case the executables can't be found via the paths defined in the `PATH` env var), and a list of command line arguments which the user can toggle via the GUI instead of having to define them manually.
https://github.com/streamlink/streamlink-twitch-gui/blob/master/src/config/players.json

JRiver Media Center only lists these command line parameters in their wiki, which are not useful for Streamlink:
https://wiki.jriver.com/index.php/The_Command_Line

If your player uses different executable names for each version, then create a symbolic link on your file system that points to the current executable name. That even works on Windows and NTFS. A player profile would be no use here.
